canonical-web-and-design/jaas-dashboard
# Convert InfoPanel and ModelDetails to use new modelData selector

## Done

Convert InfoPanel and ModelDetails to use new modelData selector

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The model details page should work as expected.

## Details

fixes #108
